scripts/ci/connect-test-matrix-generator.js

```diff
@@ -85,7 +85,8 @@ const daily = {
 };
 
 const legacyCanaryFirmware = {
-    firmwares: ['2.3.0', '2-main'],
+    // TODO(karliatto): is it possible that if we make '2.3.0' legacy results are not working properly?
+    firmwares: ['2.2.0', '2-main'],
```

Comment:
these test should work for any firmware, be it 2.2.0 or 2.3.0, doesn't matter. if we had time and resources it would be nice to run the test against all firmwares that ever released. 

if it fails for 2.3.0, it can be either wrong fixtures or some bug in the runtime.

Reply:
With this comment I just wanted to use it for debugging why the tests are failing.
Well, if I understand well removing support means that we are not going to have specific code to solve logic from older versions. So if we do not support anymore older versions, it does not make sense to me testing them. 
But I might be confused.